File: RayCast/Assets/Standard Assets/Character Controllers/Sources/Scripts/AnimationController.cs
using UnityEngine;
using System.Collections;

public class AnimationController : MonoBehaviour {

	Animator animator;
	bool grounded = true;
	
	// Use this for initialization
	void Start () {
		animator = GetComponent<Animator>();
	}
	
	// Update is called once per frame
	void Update () {
		if (grounded) {
			float h = Input.GetAxis ("Horizontal");
			float v = Input.GetAxis ("Vertical");

			animator.SetBool("Grounded", true);
			animator.SetFloat ("Forward", v);
			animator.SetFloat ("Side", h);
		} else {
			// magic fairy here for jump

			animator.SetBool("Grounded", false);
			animator.SetFloat ("Forward", 0);
			animator.SetFloat ("Side", 0);
		}
	}
}

File: RayCast/Assets/Standard Assets/Character Controllers/Sources/Scripts/AnimationController.js
#pragma strict
var animator : Animator;
var characterMotor : CharacterMotor;

function Start () {
	animator = GetComponentInChildren(Animator);
	characterMotor = GetComponent(CharacterMotor);
}

function Update () {
	if (animator != null) {
		animator.SetBool ("Grounded", characterMotor.grounded);
		animator.SetFloat ("Forward", Input.GetAxis ("Vertical"));
		animator.SetFloat ("Side", Input.GetAxis ("Horizontal"));
	}
}